src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,91 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.lang.Integer.parseInt;
+
+public class StringAddCalculator {
+    public static final String REGEX = ",|:";
+    public static final String CUSTOM_REGEX = "//(.)\n(.*)";
+    public static final int DELIMITER_GROUP = 1;
+    public static final int TEXT_GROUP = 2;
```

Comment:
```suggestion
    private static final String REGEX = ",|:";
    private static final String CUSTOM_REGEX = "//(.)\n(.*)";
    private static final int DELIMITER_GROUP = 1;
    private static final int TEXT_GROUP = 2;
```
상수 활용 👍 

외부에 노출되지 않는 상수는 private으로 막아주면 좋겠네요 :)